cursorless-talon-dev/src/cursorless_dev_vscode.talon

```diff
@@ -1,18 +1,25 @@
 app: vscode
+win.title: /cursorless/
 -
 ^install local$:
-    user.vscode_with_plugin("workbench.action.tasks.runTask", "Install local")
+    user.run_rpc_command("workbench.action.tasks.runTask", "Install local")
+^uninstall local$:
+    user.run_rpc_command("workbench.action.tasks.runTask", "Uninstall local")
 ^pre commit run$:
-    user.vscode_with_plugin("workbench.action.tasks.runTask", "Run pre commit")
+    user.run_rpc_command("workbench.action.tasks.runTask", "Run pre commit")
 
 ^serve start$:
-    user.vscode_with_plugin("workbench.action.tasks.runTask", "Serve cursorless.org")
+    user.run_rpc_command("workbench.action.tasks.runTask", "Serve cursorless.org")
 ^serve stop$:
-    user.vscode("workbench.action.tasks.terminate")
+    user.run_rpc_command("workbench.action.tasks.terminate")
     insert("Serve")
     key(enter)
 
 ^debug generate subset$:
-    user.vscode_with_plugin("workbench.action.tasks.runTask", "Generate test subset file")
+    user.run_rpc_command("workbench.action.tasks.runTask", "Generate test subset file")
 ^debug edit subset$:
-    user.vscode_with_plugin("commands.openFolder", "./packages/test-harness/testSubsetGrep.properties")
+    user.run_rpc_command("commands.openFolder", "./packages/test-harness/testSubsetGrep.properties")
+
+debug {user.cursorless_launch_configuration}:
+    user.run_rpc_command("commands.startDebugging", cursorless_launch_configuration)
+    user.run_rpc_command("workbench.debug.action.focusRepl")
```

Comment:
I find I want to see the debug output 95% of the time so this saves me saying "debug console" after "debug test"

Reply:
i find that 95% of the time after starting the extension, i want to switch to that window. unfortunately, doing that is pretty non-portable (afaik).

Reply:
What do you think about hiding the repl on stench stop?

Reply:
> i find that 95% of the time after starting the extension, i want to switch to that window. unfortunately, doing that is pretty non-portable (afaik).

For me that happens automatically I believe. Is that not the case for you?

Reply:
> What do you think about hiding the repl on stench stop?

Huh I don't have a strong feeling. Does the debug output get clobbered when you stop? Or is that just when you start another run? If it gets clobbered then we should def hide; if not I could go either way

Reply:
When you start a new I believe

Reply:
I don't know how to hide the repl on stopping extension. I'd be inclined to punt on that anyway and get this PR in